### ISSUE TITLE
dropdowns not populating

### ISSUE THREAD

**Issue by alnipper49** [alexanderzobnin/grafana-zabbix#459]
### I'm submitting a ...
- [x ] Bug report
- [ ] Feature requestdd
- [ ] Question / Support request

_If you don't send bug report, remove sections below_
### For bug report please include this information:

**Software Versions**

| Grafana | Zabbix | Grafana-Zabbix Plugin |
| ------- | ------ | --------------------- |
| 4.4.3   | 3.4.1  | most recent              |

#### What did you do?
everything installed - both plugin and data source are not reporting issues.  

#### What was the expected result?
When clicking on groups I was expecting to get a list of all of the groupings created and in use on the zabbix server

#### What happened instead?
When adding items to the dashboard instead of a dropdown of items to pick from I'm getting variables.  (see below).  Even manually typing in what they (should) be isn't working.

Even though the data source is all green I've tried all the various permutations.  Enclosed are how they are set now, though like I said above I've probably tried 30+ different combos.  
![capture](https://user-images.githubusercontent.com/8738171/30230513-98b14ade-94b4-11e7-8aae-baaab0a22098.PNG)

If it relates to metric data visualization would be great to get
- An image or text representation of your metric query
- The raw query and response for the network request (check this in chrome dev tools network tab, here you can see metric requests and other request, please include the request body and request response)
![capture](https://user-images.githubusercontent.com/8738171/30230428-43366454-94b4-11e7-95e7-f0d8b6ee821a.PNG)

**Comment by alexanderzobnin** [comment]
@alnipper49 can you show grafana backend log?

**Comment by alnipper49** [comment]
That's the weird thing.  My grafana.log doesn't have any logging with regards to the plugin making any calls into zabbix.  I'm smashing the test button and don't see much to indicate issues.

`t=2017-09-06T11:26:14-0400 lvl=info msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=GET path=/api/status.json status=404 remote_addr="10.222.222.222, 127.0.0.1" time_ms=182 size=35 referer=http://10.222.222.211/datasources/edit/9
t=2017-09-06T11:26:14-0400 lvl=info msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=GET path=/api/status.json status=404 remote_addr="10.222.222.222, 127.0.0.1" time_ms=179 size=35 referer=http://10.222.222.211/datasources/edit/9
t=2017-09-06T11:26:14-0400 lvl=info msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=GET path=/api/status.json status=404 remote_addr="10.222.222.222, 127.0.0.1" time_ms=284 size=35 referer=http://10.222.222.211/datasources/edit/9
t=2017-09-06T11:26:15-0400 lvl=info msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=GET path=/api/status.json status=404 remote_addr="10.222.222.222, 127.0.0.1" time_ms=181 size=35 referer=http://10.222.222.211/datasources/edit/9
t=2017-09-06T11:26:17-0400 lvl=info msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=GET path=/api/status.json status=404 remote_addr="10.222.222.222, 127.0.0.1" time_ms=215 size=35 referer=http://10.222.222.211/datasources/edit/9
t=2017-09-06T11:26:18-0400 lvl=info msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=GET path=/api/status.json status=404 remote_addr="10.222.222.222, 127.0.0.1" time_ms=180 size=35 referer=http://10.222.222.211/datasources/edit/9
t=2017-09-06T11:27:39-0400 lvl=info msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=GET path=/api/status.json status=404 remote_addr="10.222.222.222, 127.0.0.1" time_ms=186 size=35 referer=http://10.222.222.211/datasources/edit/9
t=2017-09-06T14:12:58-0400 lvl=info msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/opennms/rest/measurements status=502 remote_addr="10.222.222.41, 127.0.0.1" time_ms=51 size=0 referer="http://10.222.222.211/dashboard/db/gvl?refresh=5m&orgId=1"
t=2017-09-06T15:17:20-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=363 size=516 referer="http://10.222.222.211/dashboard/db/sosh?panelId=1&fullscreen&edit&orgId=1"
t=2017-09-06T15:17:28-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=267 size=517 referer="http://10.222.222.211/dashboard/db/sosh?panelId=1&fullscreen&edit&orgId=1"
t=2017-09-06T15:23:16-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=256 size=907 referer="http://10.222.222.211/dashboard/db/sosh?panelId=1&fullscreen&edit&orgId=1&tab=metrics"
t=2017-09-06T15:41:35-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=302 size=849 referer="http://10.222.222.211/dashboard/db/sosh?panelId=1&fullscreen&edit&orgId=1&tab=metrics"
t=2017-09-06T15:43:30-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=316 size=851 referer="http://10.222.222.211/dashboard/db/sosh?panelId=1&fullscreen&edit&orgId=1&tab=metrics"
t=2017-09-06T15:43:47-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=263 size=857 referer="http://10.222.222.211/dashboard/db/sosh?panelId=1&fullscreen&edit&orgId=1&tab=metrics"
t=2017-09-06T15:45:03-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=393 size=707 referer="http://10.222.222.211/dashboard/db/sosh?panelId=1&fullscreen&edit&orgId=1&tab=metrics"
t=2017-09-06T15:45:11-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=260 size=421 referer="http://10.222.222.211/dashboard/db/sosh?panelId=1&fullscreen&edit&orgId=1&tab=metrics"
t=2017-09-06T15:45:33-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=276 size=421 referer="http://10.222.222.211/dashboard/db/sosh?panelId=1&fullscreen&edit&orgId=1&tab=general"
t=2017-09-06T15:46:45-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=381 size=421 referer="http://10.222.222.211/dashboard/db/sosh?orgId=1"
t=2017-09-06T15:47:03-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=43 size=425 referer="http://10.222.222.211/dashboard/db/sosh?panelId=1&fullscreen&edit&orgId=1"
t=2017-09-06T15:47:32-0400 lvl=info msg=type logger=tsdb.mysql type="&{name:1 hasNullable:false hasLength:false hasPrecisionScale:false nullable:false length:0 databaseType:LONGLONG precision:0 scale:0 scanType:0xfc3240}"
t=2017-09-06T15:47:39-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=265 size=421 referer="http://10.222.222.211/dashboard/db/sosh?orgId=1"
t=2017-09-06T15:47:49-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=258 size=421 referer="http://10.222.222.211/dashboard/db/sosh?panelId=1&fullscreen&edit&orgId=1"
t=2017-09-06T15:47:56-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=24 size=425 referer="http://10.222.222.211/dashboard/db/sosh?panelId=1&fullscreen&edit&orgId=1"
t=2017-09-06T15:49:43-0400 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/tsdb/query status=500 remote_addr=10.222.222.222 time_ms=83 size=330 referer="http://10.222.222.211/dashboard/db/sosh?panelId=1&fullscreen&edit&orgId=1"
`

**Comment by alnipper49** [comment]
(10.222.222.222 is my workstation, 211 is the zabbix server)

**Comment by alexanderzobnin** [comment]
I think it's related to #445 too. Try to disable ONLY_FULL_GROUP_BY in MySQL or disable Direct DB Connection until next release.

**Comment by alnipper49** [comment]
Disabled database access - not getting ANY drop downs now.  I'll have to poke around some more, looking at your bugs this is so far away from anything else my gut is telling me it's something on the zabbix end at this point.

**Comment by alexanderzobnin** [comment]
Check user's permissions in Zabbix.

**Comment by alnipper49** [comment]
That's it!  It had to be super admin.  It was in the administrators group, but user type was the key.

You're the best!

**Comment by alexanderzobnin** [comment]
I recommend using read-only user due to security reason. Just ensure user has access to api and needed hosts.

**Comment by thebeerman** [comment]
I had a similar problem where when I would click in the Group feild, all I would see was Templates/Operating Systems or Zabbix Servers.  Nothing I changed on my api user permissions in zabbix seemed to fix.  I then edited dashboard in Firefox (was having issue in Chrome) and Firefox was working correctly.
I did the Empty Cache and Hard reload in Chrome for the Grafana server, and that fixed the issue.
